src/components/ComputeModal.js

```diff
@@ -547,7 +547,17 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
 
   const makeImageInfo = style => div({ style: { whiteSpace: 'pre', ...style } }, [
     div({ style: Style.proportionalNumbers }, ['Updated: ', updated ? Utils.makeStandardDate(updated) : null]),
-    div(['Version: ', version || null])
+    div([h(isTerraSupported(_.find({ image: selectedLeoImage }, leoImages)) ? Link : span,
+      {
+        href: getChangelogUrl(_.find({ image: selectedLeoImage }, leoImages)),
+        ...Utils.newTabLinkProps
+      }, [
+        'Version: ', version || null
+      ]), h(ClipboardButton, {
+      text: selectedLeoImage,
+      style: { marginLeft: '0.5rem' },
+      tooltip: copyImageLinkTooltipText
+    })])
```

Comment:
- `h(` needs to be followed by React components (whose initials are capitalized for distinction).
- I wonder if we could just disable the `Link` for the case of community-maintained images.
- Could remove the outer `div`.

```suggestion
    h(Link, {
      disabled: !isTerraSupported(_.find({ image: selectedLeoImage }, leoImages)),
      href: getChangelogUrl(_.find({ image: selectedLeoImage }, leoImages)),
      ...Utils.newTabLinkProps
    }, ['Version: ', version || null]),
    h(ClipboardButton, {
      text: selectedLeoImage,
      style: { marginLeft: '0.5rem' },
      tooltip: copyImageLinkTooltipText
    })
```

Feel free to run the `disabled` `Link` change by UX.